maker/src/actor_system.rs

```diff
@@ -164,10 +167,33 @@ where
         );
         let listener_supervisor = supervisor.create(None).spawn(&mut tasks);
 
-        let (supervisor, _ping_address) = supervisor::Actor::new({
+        let (supervisor, ping_address) = supervisor::Actor::new({
+            let endpoint_addr = endpoint_addr.clone();
             move || ping::Actor::new(endpoint_addr.clone(), PING_INTERVAL)
         });
-        let _ping_supervisor = supervisor.create(None).spawn(&mut tasks);
+        let ping_supervisor = supervisor.create(None).spawn(&mut tasks);
+
+        let (supervisor, _connection) = supervisor::Actor::with_policy(
+            {
+                move || {
+                    endpoint_monitor::Actor::new(
+                        endpoint_addr.clone(),
+                        endpoint_monitor::Subscribers::new(
+                            vec![xtra::message_channel::MessageChannel::clone_channel(
+                                &ping_address,
+                            )],
+                            vec![xtra::message_channel::MessageChannel::clone_channel(
+                                &ping_address,
+                            )],
+                            vec![],
```

Comment:
⚠️ @klochowicz isn't the listener actor missing here?

Reply:
thanks for spotting, preparing a fixup right now!